src/domain/model/test/interpretation/InterpretationRules.test.ts

```diff
@@ -0,0 +1,66 @@
+import { InterpretationRules } from './InterpretationRules';
+import { InterpretationTheme } from './Interpretation';
+import { Results } from '../Results';
+import { UserId } from '../../user/UserId';
+import { ConfidenceLevel } from '../ConfidenceLevel';
+
+describe('InterpretationRules', () => {
+  it('produces a correct Interpretation when the condition evaluates to true', () => {
+    const interpretationRule = InterpretationRules.from([
+      {
+        output: {
+          namePattern: 'Some pattern {{value}}',
+          theme: InterpretationTheme.NEGATIVE,
+          propertyVariables: {
+            value: 'c',
+          },
+        },
+        condition: {
+          property: 'c',
+          comparator: '>',
+          value: 2,
+        },
+      },
+    ]);
+
+    const results = new Results(new UserId(), { c: 3 }, ConfidenceLevel.LOW);
+    const interpretations = interpretationRule.interpret(results);
+
+    expect(interpretations[0]?.name).toEqual('Some pattern {{value}}');
+    expect(interpretations[0]?.theme).toEqual(InterpretationTheme.NEGATIVE);
+    expect(interpretations[0]?.variables.get('value')).toEqual(3);
```

Comment:
So you're going to have the client substitute the value, or did I misunderstand?

Reply:
Yes. the reason is to allow for localisation in the future.